### ISSUE TITLE
How can I use the test script?

### ISSUE THREAD

**Issue by nightskydev** [valekar/nft-vault#4]
Dear valekar.
I want to know how to use the test script.

**Comment by valekar** [comment]
@nightfury1111111111  Yes, I will update the readme, in meanwhile, check this resource for running test cases - ### https://book.anchor-lang.com/getting_started/hello_anchor.html
